src/dsymbol.d

```diff
@@ -819,6 +810,15 @@ public:
         return p ? p.isAggregateDeclaration() : null;
     }
 
+    // is this a member of an AggregateDeclaration?
```

Comment:
`Dsymbol.isMember2` is now a tweaked version of `Dsymbol.isMember`. Moving the function is good to represent those relations.